src/components/vertical-nav.tsx

```diff
@@ -0,0 +1,58 @@
+import { Link, useLocation } from "@tanstack/react-router";
+import { cn } from "../lib/utils";
+import SimpleBar from "simplebar-react";
+import "simplebar-react/dist/simplebar.min.css";
+
+const NavLink = ({ link, title }: { link: string; title: string }) => {
+  const currentRoute = useLocation();
+  const isActive = (path: string) => currentRoute.pathname === path;
+
+  return (
+    <Link
+      to={link}
+      className={cn("my-3 text-xl font-semibold hover:text-[#EAD8B1]", {
+        "text-[#EAD8B1]": isActive(link),
+      })}
+    >
+      {title}
+    </Link>
+  );
+};
+
+const navLinks = [
+  { link: "/dashboard", title: "Dashboard" },
+  { link: "/resume", title: "Resume" },
+  { link: "/applications", title: "Applications" },
+  { link: "/find-jobs", title: "Find Jobs" },
+];
+
+export default function VerticalNav() {
+  return (
+    <SimpleBar className="h-screen w-[300px] fixed bg-[#001F3F] text-white overflow-auto flex flex-col p-7">
+      <div className="flex items-center justify-between">
+        <img src="/logo-nav.svg" alt="logo" className="w-[100px] h-[100px]" />
+        <h1 className="font-bold text-2xl">WORKGO</h1>
+      </div>
+      <div className="flex flex-col mt-10">
+        {navLinks.map((navLink) => (
```

Comment:
SimpleBar should wrap only this map instead of the entire sidebar

Reply:
This is because, in smaller screens, the scrollbar would hide the user information. That is less than ideal. We would rather prefer to hide half of nav links instead.

Reply:
That makes more sense. Done